src/coord/src/catalog/builtin.rs

```diff
@@ -235,6 +235,14 @@ lazy_static! {
         id: GlobalId::System(2009),
         index_id: GlobalId::System(2010),
     };
+    pub static ref MZ_DATABASE: BuiltinTable = BuiltinTable {
```

Comment:
I know we're modeling this after `pg_database` (which is singular), but I think it's more important to follow the `mz_catalog` convention here, which is to use plural table names—so `mz_databases`!